scaling_performance/host_practices.adoc

```diff
@@ -57,42 +57,94 @@ include writing a large amount of log data.
 etcd is a distributed key-value store that {product-title} uses for
 configuration.
 
+Version 3 of etcd brings with it important scalability and performance improvements that reduce CPU, memory, network and disk requirements for any size cluster.  OpenShift 3.5 introduced etcd version 3.1.
```

Comment:
This sentence should clarify and establish terminology for the rest:

1. release version of etcd (3.x)
2. storage schema (v2 vs v3)

and clearly use it.  This is very confusing for people.  I would recommend saying "etcd 3.x" and "etcd storage v2" and "etcd storage v3".